webui-ros-joystick/launch/panther_webui.launch

```diff
@@ -2,10 +2,11 @@
 
   <arg name="node_name" default="node_server" />
   <arg name="wait_nodes" default="" />
+  <arg name="cmd_vel_topic" default="/cmd_vel" />
   <arg name="max_lin_vel" default="2.0" />
   <arg name="max_ang_vel" default="1.0" />
-  <arg name="max_lin_accel" default="2.0" />
-  <arg name="max_ang_accel" default="2.0" />
+  <arg name="max_lin_accel" default="20.0" />
```

Comment:
Was this intentional?

Reply:
Yes, that should be default configuration but was missed in previous pr

Reply:
Generally we should do things like this in separate PRs